src/CoreBundle/private/modules/CMSTemplateEngine/CMSTemplateEngine.class.php

```diff
@@ -475,12 +475,26 @@ protected function _GetModuleInstanceList()
      */
     protected function filterMainNavigation(): void
     {
-        $this->data['oMenuItems'] = $this->oTableManager->oTableEditor->GetMenuItems();
-        $this->data['oMenuItems']->RemoveItem('sItemKey', 'save');
-        $this->data['oMenuItems']->RemoveItem('sItemKey', 'copy');
-        $this->data['oMenuItems']->RemoveItem('sItemKey', 'new');
-        $this->data['oMenuItems']->RemoveItem('sItemKey', 'delete');
-        $this->data['oMenuItems']->RemoveItem('sItemKey', 'copy_translation');
+        /**
+         * @var $menuItems TIterator
+         */
+        $menuItems = $this->oTableManager->oTableEditor->GetMenuItems();
+
+        $menuItems->RemoveItem('sItemKey', 'save');
+        $menuItems->RemoveItem('sItemKey', 'copy');
+        $menuItems->RemoveItem('sItemKey', 'new');
+        $menuItems->RemoveItem('sItemKey', 'delete');
+        $menuItems->RemoveItem('sItemKey', 'copy_translation');
+
+        $tableEditorButton = $menuItems->FindItemWithProperty('sItemKey', 'edittableconf');
+        if (false !== $tableEditorButton) {
+            /**
```

Comment:
The type docblock needs to move to line 488 where the variable is introduced

Reply:
Only here the type is safe, two lines earlier it could also be a boolean.

(And it is only needed/meaningful because FindItemWithProperty() is annotated with "object".)